provisioning/roles/php-fpm/tasks/main.yml

```diff
@@ -1,3 +1,10 @@
 - name: install php5-fpm
   apt: pkg=php5-fpm state=latest
-  sudo: yes
+  sudo: yes  
+  when: "{{ php_version | float }} < 7.0"
```

Comment:
You can also use a `ternary` here probably :

   {{ (php_version | float == 7.0) | ternary('php7.0-fpm', 'php5-fpm') }}